### DISCUSSION TITLE
'BoolSelectDropdown'. Did you mean: 'MultiSelectDropdown'? error, gitpod, local docker (windows and linux),

### DISCUSSION THREAD

**Discussion by szymontex** [omkarcloud/google-maps-scraper#185] [Q&A]
gitpod /workspace/google-maps-scraper (master) $ python run.py
Traceback (most recent call last):
  File "/workspace/google-maps-scraper/backend/scrapers.py", line 293, in <module>
    filters.BoolSelectDropdown("website", prioritize_no=True),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'botasaurus_server.filters_export' has no attribute 'BoolSelectDropdown'. Did you mean: 'MultiSelectDropdown'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/google-maps-scraper/run.py", line 1, in <module>
    import backend.scrapers
  File "/workspace/google-maps-scraper/backend/scrapers.py", line 325, in <module>
    filters.BoolSelectDropdown("website", prioritize_no=True),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'botasaurus_server.filters_export' has no attribute 'BoolSelectDropdown'. Did you mean: 'MultiSelectDropdown'?

 > [bot-1 7/7] RUN python run.py install:
1.498 Downloading @request dependencies...
5.937 100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.8/13.8 MB 4.5 MB/s
6.605 Traceback (most recent call last):
6.605   File "/app/backend/scrapers.py", line 293, in <module>
6.605     filters.BoolSelectDropdown("website", prioritize_no=True),
6.605     ^^^^^^^^^^^^^^^^^^^^^^^^^^
6.605 AttributeError: module 'botasaurus_server.filters_export' has no attribute 'BoolSelectDropdown'. Did you mean: 'MultiSelectDropdown'?
6.605
6.605 During handling of the above exception, another exception occurred:
6.605
6.605 Traceback (most recent call last):
6.605   File "/app/run.py", line 1, in <module>
6.606     import backend.scrapers
6.606   File "/app/backend/scrapers.py", line 325, in <module>
6.606     filters.BoolSelectDropdown("website", prioritize_no=True),
6.606     ^^^^^^^^^^^^^^^^^^^^^^^^^^
6.606 AttributeError: module 'botasaurus_server.filters_export' has no attribute 'BoolSelectDropdown'. Did you mean: 'MultiSelectDropdown'?
------
failed to solve: process "/bin/sh -c python run.py install" did not complete successfully: exit code: 1

**Comment by Chetan11-dev** [comment]
please update it following https://github.com/omkarcloud/google-maps-scraper/blob/master/advanced.md#-how-to-update-the-scraper-to-the-latest-version
